website/docs/recipes/using-pnpm-with-lerna.md

```diff
@@ -5,7 +5,7 @@ Lerna can be used in a [`pnpm` workspace](https://pnpm.io/workspaces) to get the
 When used in a `pnpm` workspace, Lerna will:
 
 - resolve package locations with `pnpm-workspace.yaml` (https://pnpm.io/workspaces)
-- ignore `"workspaces"` in `package.json`
+- enforce `useWorkspaces: true` in `lerna.json` (and ignore `packages:` in `package.json`).
```

Comment:
This change doesn't make sense, please revert. There is no packages property in package.json, there is a workspaces property as was stated before

Reply:
The Main Job has been failing since April 5. This PR was a test to rule out the possibility, however small, that this change made on April 4 contributed to it. As expected, it did not, so will close; the actual PR to upgrade tar is https://github.com/lerna/lerna/pull/3990

Reply:
Yeah the issue seems to be pnpm v9